franklin/builder/serializers.py

```diff
@@ -27,6 +27,18 @@ class Meta:
         }
 
 
+class EnvironmentStatusSerializer(serializers.ModelSerializer):
```

Comment:
I'm not sure best practices, if you you are supposed to have one Serializer for one model, or different Serializers for different functions.  I have seen a mixture of both.